python_scripts/tests/api_tests/database_api/request_template.json

```diff
@@ -0,0 +1,6 @@
+{
+    "jsonrpc": "2.0",
+    "id": $test_id,
+    "method": "call",
```

Comment:
Change this to the new call format assuming the replacement works.

```
"method":"$api.$method"
"params":$args
```

Reply:
done